src/generator.cs

```diff
@@ -3454,8 +3460,8 @@ public string FormatTypeUsedIn (string usedInNamespace, Type type, bool protocol
 			return "nint";
 		if (type == TypeManager.System_nuint)
 			return "nuint";
-		if (type == TypeManager.System_Boolean)
```

Comment:
It looks like I'm removing a case for boolean, but it's already handled above (the code was duplicated before).